src/features/collections/MatchPane.tsx

```diff
@@ -34,7 +38,13 @@ const ViewMatch = () => {
     (state) => state.collections.currentSelection.length
   );
 
-  const matchQuery = usePollMatch(matchId);
+  const matchQuery = getMatch.useQuery(matchId || '', {
+    skip: !matchId,
+  });
+  useBackoffPolling(
+    matchQuery,
+    (result) => !!(result.error || result.data?.state !== 'processing')
```

Comment:
> Prefer Number/Boolean in new code

-- us, maybe, 2023-03-21

Reply:
I'm going to add a "readable boolean logic" section to the europa meeting notes